### PR TITLE
Playground launcher

### PR DESCRIPTION
**Pull request by AlveLarsson**
Adds an Editor toolbar button to build and launch the playground

Note:
This is currently limited due to security issues with browsers and filesystems, a future workaround will be considered

image

### CODE REVIEW

File: Assets/Editor/MoNA/SendToPlayground.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        {
+            GUILayout.FlexibleSpace();
+
+            GUI.skin.button.fontSize = 13;
+            if (GUILayout.Button(new GUIContent("▶️ Playground", "Build and open playground")))
+            {
+                Helpers.UpsertExportsDirectory();
+                BuildPipeline.BuildAssetBundles(Constants.PlaygroundDirectory, BuildAssetBundleOptions.None, BuildTarget.WebGL);
+                Helpers.OpenDirectory(Constants.ExportsDirectory);
+                Application.OpenURL("https://www.mona.gallery/playground");
```
Comment (theAlexPatin) [inline Assets/Editor/MoNA/SendToPlayground.cs:26]: Could you create a reference to this in the `Constants` - this URL may be subject to change so just easier to track down that way :)

**Review by theAlexPatin — APPROVED** [review]
Super helpful, thanks Alve! Just the one comment and then this should be ready to merge.

File: Assets/Editor/MoNA/SendToPlayground.cs (hunk 1 above)
Comment (AlveLarsson) [reply Assets/Editor/MoNA/SendToPlayground.cs]: Got it, changed it + formating

**Comment by AlveLarsson** [comment]
@theAlexPatin Ready to merge

**Pull request merged**